notebooks/official/tabular_workflows/prophet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1291 @@
+{
```

Comment:
<p>Do we want to mention data aggregation by time series ID prior to batch prediction? And "Post processing to disaggregate batch prediction results to look like Vertex Forecasting's output"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1320/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
Added to the beginning of the section.